app/callbacks/data_callbacks.py

```diff
@@ -145,35 +145,31 @@ def api_watcher(n_intervals, _, local_events, local_alerts, user_headers, user_c
         if len(v) == 0:
             raise PreventUpdate
 
-        api_alerts = pd.concat(v.values).groupby(["event_id"]).head(cfg.MAX_ALERTS_BY_EVENT).reset_index(drop=True)
+        api_alerts = pd.concat(v.values).groupby(["event_id"]).head(cfg.MAX_ALERTS_PER_EVENT).reset_index(drop=True)
         new_api_alerts = api_alerts[~api_alerts["id"].isin(local_alerts["id"])].copy()
-        new_api_alerts["processed_loc"] = [process_bbox(loc) for loc in new_api_alerts["localization"]]
+        local_alerts["processed_loc"].apply(process_bbox)
```

Comment:
looks like the result of the calculation is not being used

Reply:
done !